app/controllers/api/v1/sessions_controller.rb

```diff
@@ -1,10 +1,12 @@
 module Api
   module V1
     class SessionsController < ApplicationController
+      before_action :skip_authorization
```

Comment:
Is skip_authorization the same as just not having anything in the before_action?
If the skip_authorization wasn't added, wouldn't we be able to access this controller at any time, which is what we want since we're using this to help our authentication process?

Reply:
We include Pundit via **ApplicationController** `include Pundit::Authorization`, which is inherited in every child controller.  Pundit automatically checks for authorization, based on users role, for every single controller action.

Since **SessionsController**'s SRP is creating a session and generating a token, it should not overlap with authorization at all, just authentication.  So it makes sense to put Pundit's `skip_authorization` method in **SessionsController** so that Pundit won't verify authorization in a place it shouldn't/does not exist yet.  

Simply put, authorization checks are irrelevant in this controller b/c the user is technically not even authenticated yet. Pundit will throw an error if `before_action :skip_authorization` isn't included in this particular controller.
`Pundit::AuthorizationNotPerformedError:
       Api::V1::CompaniesController`

To extrapolate further, we should not even create a policy for **SessionsController**.

Does that answer the question?